ct/happier.sh

```
@@ -1,5 +1,7 @@
 #!/usr/bin/env bash
-source <(curl -fsSL https://raw.githubusercontent.com/community-scripts/ProxmoxVE/main/misc/build.func)
+INSTALLER_REPO="${INSTALLER_REPO:-happier-dev/ProxmoxVE}"
+INSTALLER_REF="${INSTALLER_REF:-main}"
+source <(curl -fsSL "https://raw.githubusercontent.com/${INSTALLER_REPO}/${INSTALLER_REF}/misc/build.func")
```

Shellcheck (0.11.0)

[warning] ShellCheck can't follow non-constant source. Use a directive to specify location.

(SC1090)